.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+## Pull request checklist
+
+Please check if your PR fulfills the following requirements:
+
+- [ ] Tests for the changes have been added (for bug fixes / features)
+- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
+- [ ] Build (`npm run build`) was run locally for affected output targets
+
+## Pull request type
+
+<!-- Please do not submit updates to dependencies unless it fixes an issue. -->
+
+<!-- Please try to limit your pull request to one type, submit multiple pull requests if needed. -->
+
+Please check the type of change your PR introduces:
+
+- [ ] Bugfix
+- [ ] Feature
+- [ ] Code style update (formatting, renaming)
+- [ ] Refactoring (no functional changes, no api changes)
+- [ ] Build related changes
+- [ ] Documentation content changes
+- [ ] Other (please describe):
+
+## What is the current behavior?
+
+<!-- Please describe the current behavior that you are modifying. -->
+
+<!-- Issues are required for both bug fixes and features. -->
+
+Issue #:
+
+## What is the new behavior?
+
+<!-- Please describe the behavior or changes that are being added by this PR. -->
+
+-
```

Comment:
No action required - FWIW I had been considering changing this to an open text area.

IIRC The original intent was to ensure that folks filled this section out, with the thought
being if we stubbed this section it would encourage people opening PRs to add content
(instead of leaving an empty section).

In practice I've found (personally and in the community) is that folks just delete these and
write a couple sentences.

Reply:
If people do that, we will close the PR without review. In Framework we have had the opposite experience (people will fill out the template). I thing a strong hand drives better quality here. If people are unwilling to fill out a 15s template, we are unwilling to review their contribution.